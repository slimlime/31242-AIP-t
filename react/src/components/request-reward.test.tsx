import React from "react";
import { shallow } from "enzyme";
import RequestReward from "./request-reward";

describe("<Request />", () => {
  it("should render correctly", () => {
    const wrapper = shallow(
      <RequestReward
        itemId="201"
        display_name="Coffee"
        setSelectedReward={() => console.log("Passed")}
      />
    );
    expect(wrapper).toMatchSnapshot();
  });
});
