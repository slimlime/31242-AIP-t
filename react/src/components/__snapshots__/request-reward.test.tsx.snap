// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Request /> should render correctly 1`] = `
<ClickAwayListener
  onClickAway={[Function]}
>
  <div
    className="popUpCircle"
    onClick={[Function]}
  >
    Coffee
  </div>
</ClickAwayListener>
`;
